pages/GSoC.md

```diff
@@ -341,6 +341,17 @@ should be opened in DispVM or locally)
 
 **Mentor**:
 
+### New Qubes Manager
+**Project**: New Qubes Manager
```

Comment:
I was not aware the work was ongoing. What do you suggest?
- Include those links in project description?
- Remove project proposal as it's already being handled?